addons/frag/functions/fnc_dev_trackHitBox.sqf

```diff
@@ -0,0 +1,75 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add a hit box outline to an object, outdated for unit hits as they
+ * use hit-point locations.
+ *
+ * Arguments:
+ * 0: Object that should have it's hit box drawn. <OBJECT>
+ * 1: Add sphere at object origin. <BOOL> (Default: true)
+ *
+ * Return Value:
+ * None
```

Comment:
Aight, will do. 
I was alluding to the fact that objects that inherit from `CAManBase` use the hit points defined in `ACE_FRAG_HITPOINTS` to target fragments rather than a bounding box area.

Reply:
Ok I omitted mention of what I discussed in the above comment since it isn't relevant to the function